src/lib/rpc/provider.ts

```diff
@@ -391,12 +391,14 @@ export class JsonRpcProvider implements IJsonRpcProvider {
         return this
             .callApiMethod<Array<any[]>>(Methods.SearchDataRequests, {
                 radHash,
-                limit: options?.limit,
-                offset: options?.offset,
-                reverse: options?.reverse
+                limit: options?.limit || 1,
+                offset: options?.offset || 0,
+                reverse: options?.reverse || true
```

Comment:
If we use `reverse = false`, then `false || true` is always `true` so reverse can't never be `false`. We have to check if it's `false` instead of `undefined`